### DISCUSSION TITLE
Lightmaps only visible when material preferences are set to low

### DISCUSSION THREAD

**Discussion by TheBricktop** [mozilla/hubs#3714] [Art Questions] [answered]
Hello, testing blender hubs export: so far its awesome but i have some issues with lightmap and how they are handled in browser, the thing is they arent displayed if the materials are set to High. Same thing on pc mac and android.
I think that this is not very good because i might want to have both lightmap and high quality materials in the same room.

**Comment by j-conrad** [comment, marked as answer]
Using lightmaps was really intended to be used with unlit materials. 
But lightmaps will still work when used on a material using Blender's Principled BSDF shader. However, because the material will now accept lighting and reflections (if it's glossy) so any lightmap texture you're using will get washed out, negating the benefits of baked lighting.
If you crank up the 'Intensity' parameter of the lightmap node in Blender, you may find you can see it better. You may need to use a relatively high Intensity value depending on how bright your lighting and skybox brightness (which affects glossy reflections) are.
